package.json

```diff
@@ -1,11 +1,22 @@
 {
   "name": "mysterium-vpn-js",
-  "version": "1.0.16",
+  "version": "1.1.0",
   "description": "Javascript SDK for Mysterium node",
+  "keywords": ["mysterium", "vpn", "dvpn", "sdk"],
   "repository": {
     "type": "git",
     "url": "https://github.com/mysteriumnetwork/mysterium-vpn-js.git"
   },
+  "author": {
+    "name": "Mysterium Network",
+    "email": "team@netsys.technology",
```

Comment:
Here we should use team@mysterium.network

Reply:
I sent an email to this and I didn't receive one. Hence it doesn't fit the purpose :)

Reply:
@Waldz what email we should use in such packages? An email which will notify all dev team members. Do we have dev@mysterium.network?